eng/Versions.targets

```diff
@@ -4,6 +4,7 @@
     <PackageReference Update="EnvDTE" Version="$(EnvDTEVersion)" />
     <PackageReference Update="EnvDTE80" Version="$(EnvDTE80Version)" />
     <PackageReference Update="Microsoft.CodeAnalysis.Workspaces.Common" Version="$(MicrosoftCodeAnalysisWorkspacesCommonVersion)" />
+    <PackageReference Update="Microsoft.CodeAnalysis.ExternalAccess.EditorConfigGenerator" Version="$(MicrosoftCodeAnalysisWorkspacesCommonVersion)" />
```

Comment:
Do you want to be defining a new variable for the EditorConfigGenerator package? Or just consider using CommonMicrosoftCodeAnalysisVersion?